### ISSUE TITLE
Replicability

### ISSUE THREAD

**Issue by ludovicdmt** [VDelv/Spatio-Temporal-EEG-Analysis#1]
Hello,   

I have read recently and with great interest your latest paper "Spatio-Temporal Analysis of Transformer based Architecture for Attention Estimation from EEG".  
I tried to run your notebooks for the `phydaa` dataset (**no modifications at all**, just download the data from Figshare and put it in the folder as mentioned in the Readme). I ran the notebook with the Transformer network and the one with the Resnet.   

As depicted in the two screenshots attached from Tensorboard, I got nothing but overfiting on the train set. The validation loss and accuracy are only noises and do not follow the trends or values on the train. 
![ResNet](https://user-images.githubusercontent.com/19227268/165256207-b174f568-f2a5-4a2d-a204-4f0ec8517362.png)
![Transformer](https://user-images.githubusercontent.com/19227268/165256217-74579bf5-5504-4382-a8fa-a06a44c19cf3.png)

I'm running it with Pytorch 1.9.1 (not 1.7) and Cuda 11.3 (and not 10.2). Although, I don't think that should alter the convergence of the networks ?

Should I change something like the LR or the LR scheduler to make it converge ?

Thank you for your help.

Ludovic
